### PR TITLE
Feature/integrate js library

### PR DESCRIPTION
**Pull request by goldquest**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (milkovsky) [inline .gitignore]: what should we do with the "dist" directory? I think we should commit the minified version. What do you think?

Comment (goldquest) [inline .gitignore]: Yes, we should provide a bundled and minified version with the release.

File: js/stream/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Liveblog Stream
+
+## Bundling
+To use this library, you first have to bundle it up.
+To do that, you will need:
+- nodejs with npm
+    - Download via [http://nodejs.org]
+    - or use [nvm](https://github.com/creationix/nvm)
+- [webpack](https://webpack.github.io/docs/)
+    - [install](https://webpack.github.io/docs/installation.html) with `npm install webpack -g`
```
Comment (milkovsky) [inline js/stream/README.md]: I see. You can ignore this comment. I am not sure as well.

**Comment by milkovsky** [comment]
you can merge the PR right after solving the comments.

**Pull request merged**